src/hooks/useIsEligibleForRefund.ts

```diff
@@ -0,0 +1,17 @@
+import {useMemo} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+function useIsEligibleForRefund() {
+    const [account] = useOnyx(ONYXKEYS.ACCOUNT);
+
+    return useMemo(() => {
```

Comment:
It's overkill to have `useMemo` here, you are just checking the property and returning a boolean, there are no complex operations here.

Reply:
Yes agreed. I think we can just use useOnyx in the component and check property.